src/pothole_detection/Potholes/generate.py

```diff
@@ -120,28 +125,32 @@ def draw_ellipse(input_path, max_potholes):
             background = background.filter(
                 ImageFilter.GaussianBlur(random.randint(50, 125)/100))
             # Scale down image to get a choppier ellipse
-            scale = random.randint(30, 60) / 100
-            mask = Image.new("L", ((int)(image.width * scale),
-                             (int)(image.height * scale)), 0)
+            # scale = random.randint(30, 60) / 100
```

Comment:
Please delete dead code

(if you want to add parts of a file to git, try `git add -p`)